HeritageHub/heritagehub-client
# Setup react app using vite

# Set Up React Project (Modern Way)

**Description:**
This pull request sets up the initial React project using Vite for faster and more efficient development. It includes the following:

1. Initialization of the React project using Vite.
2. Installation of necessary dependencies.
3. Cleanup of boilerplate files.
4. Verification that the project runs locally without errors.

**Tasks Completed:**
- [x] Initialize the React project using Vite.
- [x] Install project dependencies.
- [x] Remove unnecessary boilerplate files.
- [x] Verify the project runs locally.

**How to Test:**
1. Clone the repository.
2. Navigate to the `heritagehub-client` directory.
3. Run `npm install` to install dependencies.
4. Run `npm run dev` to start the development server.
5. Open your browser and navigate to the provided local development URL (usually `http://localhost:5173`) to verify that the project runs without errors.

**Linked Issues:**
- This PR closes #1

**Additional Notes:**
- Ensure that you have Node.js and npm installed on your machine before running the commands.
